test/integration/features/cmd-config.feature

```diff
@@ -0,0 +1,242 @@
+    | https-proxy     | https://me:pass@proxy.com:4444                                                                  | https://me:pass@proxy.com:4444                                                                  |
+    | host-only-cidr  | 192.168.0.1/0                                                                                   | 192.168.0.1/0                                                                                   |
+    | host-only-cidr  | 192.168.0.1/16                                                                                  | 192.168.0.1/16                                                                                  |
+
+  @minishift-only
+  Scenario Outline: Setting and unsetting values for iso-url key
+     When executing "minishift config set <property> "<value>"" succeeds
+     Then JSON config file "config/config.json" contains key "<property>" with value matching "<expected>"
+      And stdout of command "minishift config get <property>" is equal to "<expected>"
+      And stdout of command "minishift config view --format {{.ConfigKey}}:{{.ConfigValue}}" contains "<property>:<expected>"
+     When executing "minishift config unset <property>" succeeds
+     Then stdout of command "minishift config get <property>" is equal to "<nil>"
+      And JSON config file "config/config.json" does not have key "<property>"
+
+  Examples: Correct values for iso-url keys
+    | property        | value                                                                                           | expected                                                                                        |
+    | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
+    | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
+    | iso-url         | file://home/me/Downloads/my_handmade_b2d.iso                                                    | file://home/me/Downloads/my_handmade_b2d.iso                                                    |
+    | iso-url         | b2d                                                                                             | b2d                                                                                             |
+    | iso-url         | centos                                                                                          | centos                                                                                          |
+
+  Scenario: Unsetting non-existing key
+     When executing "minishift config unset i-do-not-exist" succeeds
```

Comment:
this succeeds? :-O
and gives error code 0 ?

Reply:
@gbraad Yes, it succeeds. For `unset` I am 50/50 if it should give exit code 0 or other. But for `get` it probably should give exit code other than 0. We should raise an issue, wdyt?